frontend/src/rest/model/Neuron.js

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable */
+/**
+ * NeuroScan
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * Contact: info@metacell.us
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ *
+ */
+
+import ApiClient from '../ApiClient';
+import Promoter from './Promoter';
+import Synapse from './Synapse';
+
+/**
+ * The Neuron model module.
+ * @module model/Neuron
+ * @version 1.0.0
+ */
+class Neuron {
+    /**
+     * Constructs a new <code>Neuron</code>.
+     * @alias module:model/Neuron
+     * @param id {String} 
+     * @param uid {String} 
+     */
+    constructor(id, uid) { 
+        
+        Neuron.initialize(this, id, uid);
+    }
+
+    /**
+     * Initializes the fields of this object.
+     * This method is used by the constructors of any subclasses, in order to implement multiple inheritance (mix-ins).
+     * Only for internal use.
+     */
+    static initialize(obj, id, uid) { 
+        obj['id'] = id;
+        obj['uid'] = uid;
+    }
+
+    /**
+     * Constructs a <code>Neuron</code> from a plain JavaScript object, optionally creating a new instance.
+     * Copies all relevant properties from <code>data</code> to <code>obj</code> if supplied or a new instance if not.
+     * @param {Object} data The plain JavaScript object bearing properties of interest.
+     * @param {module:model/Neuron} obj Optional instance to populate.
+     * @return {module:model/Neuron} The populated <code>Neuron</code> instance.
+     */
+    static constructFromObject(data, obj) {
+        if (data) {
+            obj = obj || new Neuron();
+
+            if (data.hasOwnProperty('id')) {
+                obj['id'] = ApiClient.convertToType(data['id'], 'String');
+            }
+            if (data.hasOwnProperty('file')) {
+                obj['file'] = data['file']['url'];
```

Comment:
Had to change the generated code here and other entities (some are still missing) because the deserialization was returning [object object]